checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/api/discovery/ApiDiscoveryClient.kt

```diff
@@ -6,6 +6,13 @@ import com.worldpay.access.checkout.api.Callback
 import com.worldpay.access.checkout.logging.LoggingUtils.debugLog
 import java.util.concurrent.atomic.AtomicInteger
 
+/**
+ *[ApiDiscoveryClient] is responsible for managing the discovery of the API endpoint for a service
```

Comment:
Add space before [ApiDiscoveryClient]

Also worth mentioning that this class calls the callback with the discovered API endpoint and it utilises caching

Instead of saying "and passing the result to a [DiscoveryCache]

Reply:
updated